client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -284,6 +299,62 @@ export const Admin = () => {
     }
   }
 
+  async function updateProcessedDescriptions() {
+    console.log('Updating processed course descriptions')
+    setUpdating(true)
+    setUpdatingField("processed course descriptions")
```

Comment:
Here, it could be good to have an enum of some sort to set the updating field instead of arbitrary strings. Or use constants